src/main/java/com/lishid/openinv/listeners/OpenInvPlayerListener.java

```diff
@@ -52,18 +53,28 @@ public void onPlayerJoin(PlayerJoinEvent event) {
         }
     }
 
-    @EventHandler(priority = EventPriority.MONITOR)
```

Comment:
A better question is why was it EventPriority.MONITOR to begin with. On spigot 1.9.2, the player.dat file is saved after the PlayerQuitEvent completes. The order that was happening:

- PlayerQuitEvent (Monitor priority) -> OpenInv save inventory.
- Player saves as normal

Running in a synchronous task 1 tick later corrects the behavior to:
- Player saves as normal
- OpenInv saves